programming_guide/mini_tutorial/README.md

```diff
@@ -169,6 +169,37 @@ More on the Object FIFO data movement patterns in [Section 2b](../section-2/sect
 
 2. Modify the code in [exercise_2](./exercise_2/exercise_2.py) such that the data sizes for each worker are uneven, for example: tile_sizes = [8, 24, 16].
 
+## <ins>Runtime Sequence</ins>
+
+The arguments of the IRON runtime sequence describe buffers that will be available on the host side; the body of the sequence contains commands which describe how those buffers are moved into the AIE-array through `ObjectFifos`.
+
+```python
+data_size = 256
+data_ty = np.ndarray[(data_size,), np.dtype[np.int32]]
+
+# Dataflow with ObjectFifos
+of_in = ObjectFifo(data_ty, name="in")
+of_out = ObjectFifo(data_ty, name="out")
+
+rt = Runtime()
+with rt.sequence(tile_ty, tile_ty) as (a_in, c_out):
+    rt.start(my_worker)
+    rt.fill(of_in.prod(), a_in)
+    rt.drain(of_out.cons(), c_out, wait=True)
+```
+
+Up to five buffers are supported in the runtime sequence, where the fifth is typically used for trace support. This is further described in [Section 4b](../section-4/section-4b/README.md) of the programming guide.
+
+Runtime sequence commands are submitted to and executed by a dedicated command processor in order. The command processor will wait on commands that are set to `wait` until a token associated with their completion is generated. When all the commands in the runtime sequence have been executed the command processor sends an interrupt to the host processor.
```

Comment:
It might be good to mention `task_groups` here to optimize the grouping of waits for parallel tasks: https://github.com/Xilinx/mlir-aie/blob/main/programming_examples/basic/memcpy/README.md